src/components/projects/projects-table/projects-table/projects-table.test.tsx

```diff
@@ -116,4 +116,27 @@ describe('projects table component', () => {
     const wbsNumsAsc: HTMLElement[] = await screen.findAllByText(wbsRegex);
     expect(wbsNumsAsc.map((ele) => ele.innerHTML)).toEqual(expectedWbsOrder.reverse());
   });
+
+  it('handles sorting and reverse sorting the table by wbsNum', async () => {
```

Comment:
How is this different from other tests? For testing, the tests you want to add are ones that are going to just test the sorting function you added. Take a look at this file `src/utils/tests/validate-wbs.test.ts` to see how we're testing a function instead of a component.